pkgs/tools/misc/flashrom/default.nix

```diff
@@ -23,14 +23,17 @@ stdenv.mkDerivation rec {
     hash = "sha256-rX7htJI5xvtPj1XjZwb81zFDXbGkvS+rPYDx9yUIzO4=";
   };
 
-  patches = [
-    # Release notes for 1.4.0 state that Promontory chipsets are unsupported, and that attempting to read flash on those systems may crash the system.
-    # The patch that removes this (broken) support only made it into the 1.3.0 release, seemingly by mistake, and the relevant code has been essentially untouched since.
-    # We cherry-pick the upstream patch from 1.3.0, though amended to reference the relevant bug in the error message, rather than requesting the user email upstream.
-    # https://ticket.coreboot.org/issues/370
-    # https://review.coreboot.org/c/flashrom/+/68824
-    ./0001-sb600spi.c-Drop-Promontory-support.patch
-  ];
+  patches =
+    [
+      # Release notes for 1.4.0 state that Promontory chipsets are unsupported, and that attempting to read flash on those systems may crash the system.
+      # The patch that removes this (broken) support only made it into the 1.3.0 release, seemingly by mistake, and the relevant code has been essentially untouched since.
+      # We cherry-pick the upstream patch from 1.3.0, though amended to reference the relevant bug in the error message, rather than requesting the user email upstream.
+      # https://ticket.coreboot.org/issues/370
+      # https://review.coreboot.org/c/flashrom/+/68824
+      ./0001-sb600spi.c-Drop-Promontory-support.patch
+    ]
+    # Darwin's memcpy is a macro, and requires braced initializer lists to be parenthesized
+    ++ lib.optional stdenv.isDarwin ./0002-tests-erase-parenthesize-braced-initializer.patch;
```

Comment:
Upstream?

Reply:
Also, we try to avoid conditional patch applications where there isn't a strong reason for them to be conditional, because it makes it harder to notice when they need to be updated.  In this case, there's no harm to other platforms from applying this patch unconditionally.